### ISSUE TITLE
Strange Problem

### ISSUE THREAD

**Issue by harezhh** [mmeendez8/Autoencoder#3]
Hi, guys, 

I have a problem with the following code: 

https://github.com/mmeendez8/Autoencoder/blob/master/autoencoder.py. 

![Unbenannt](https://user-images.githubusercontent.com/60259207/73067822-ee804400-3ea9-11ea-96ee-33bfab48eac1.PNG)

The code is now running on my computer and no error messages are displayed. But the result is strange as you can see on the pictures. Once the code is finished, the results are displayed in a folder. But only the results of the 0 epoch are shown. After that nothing is displayed. I did not change anything in the code and just copied and pasted it. Can somebody help me and tell me what the problem is ? 

Furthermore only the numbers are shown but not the results of the fashion mnist.

**Comment by mmeendez8** [comment]
Which hyperparameters have you used for that execution? Have your first tested the example with the default ones?

**Comment by harezhh** [comment]
> Which hyperparameters have you used for that execution? Have your first tested the example with the default ones?

Yeah, i testes the example without any changes. Just copy and paste. But i get the strange results as you can see in the picture :/
